docs/content/developers/deploy_with_helm.md

```diff
@@ -0,0 +1,84 @@
+# Deploying onos micro-services with HELM
+
+One of the goals of the onos-config project is to provide simple deployment options
+that integrate with modern technologies. Deployment configurations can be found in
+the `/deployments/helm` folder in every repository that posses the Helm charts. 
+For example see the `onos-config/deplyments/helm` folder.
+
+## Deploying on Kubernetes with Helm
+
+[Helm] is a package manager for [Kubernetes] that allows projects to provide a
+collection of templates for all the resources needed to deploy on k8s. ONOS Config
+provides a Helm chart for deploying a cluster for development and testing. In the
+future, this chart will be extended for production use.
+
+### Resources
+
+The Helm chart provides resources for deploying the config service and accessing
+it over the network, both inside and outside the k8s cluster:
+* `Deployment` - Provides a template for ONOS Config pods
+* `ConfigMap` - Provides test configurations for the application
+* `Service` - Exposes ONOS Config to other applications on the network
+* `Secret` - Provides TLS certificates for end-to-end encryption
+* `Ingress` - Optionally provides support for external load balancing
+
+### Local Deployment Setup
+
+To deploy the Helm chart locally, install [Kind] and [Helm]. On OSX, this can be done
+using [Brew]:
+
+```bash
+> go get sigs.k8s.io/kind
```

Comment:
I would suggest to refer to kind cluster github repo for installation but if you want to keep this, use this command:
GO111MODULE="on" go get sigs.k8s.io/kind

Reply:
removed and added the link.